### DISCUSSION TITLE
See All Conversations when using Files Changed View

### DISCUSSION THREAD

**Discussion by kimostler** [community/community#11306] [Pull Requests]
When re-reviewing a PR, I usually use the the `Files Changed` view to see all the commits and all comments to see code changed as a result of a comment.  I believe this view has changed and I can't see all commits with comments and that has made me think the changes were not made.  If I select a conversation, it jumps to the comment but does not show the latest commit.  If I show all commits, the conversations are not displayed.   How can I get a view of all commits with all conversations?

**Comment by xs2bharat** [comment]
i am having same issue
